Dockerfile

```diff
@@ -31,7 +32,7 @@ RUN apt-get update && apt-get -y install software-properties-common \
         sudo \
         git \
     && make deps-ubuntu \
-    && pip install --upgrade pip setuptools \
+    && pip3 install --upgrade pip setuptools \
```

Comment:
I did try that already, that fixes this problem. But I stil fail locally to get opencv built.

Reply:
See [this comment](https://github.com/OCR-D/core/issues/983#issuecomment-1434607460)

Reply:
Upgrading to a container which is based on Ubuntu > 18.04 would also have solved the problem because then `python3-pip` installs `/usr/bin/pip`, too.